web_src/js/features/projects.js

```diff
@@ -142,4 +154,56 @@ export default async function initProject() {
       window.location.reload();
     });
   });
+
+  $(document).on('click', '#delete_board_color', function (e) {
+    e.preventDefault();
+
+    const projectHeader = $(this).closest('.board-column-header');
+
+    projectHeader.removeClass('light-label').addClass('dark-label');
+
+    $.ajax({
+      url: $(this).data('url'),
+      data: JSON.stringify({color: '#f8f8f8'}),
```

Comment:
I don't think this is deletion, deletion should set to empty color so that it is styled to theme. Also I don't think separate button is needed, just allowing to set color to empty string should be enough

Reply:
I think it is in contradiction with @6543 comment ?
> one missing pease: as per lafriks if Collor is "" no thing should be chaged & we need a way to delete the color

Reply:
It should be the same as it is in labels to have consistent UI

Reply:
Fine, so I removed this button. Now if you want to remove the color you have to let th ecolor field blank.

I also added precolor like on label form.